code/_compile_options.dm

```diff
@@ -61,7 +61,7 @@ var/list/del_counter = list()
 
 #define MIN_COMPILER_VERSION 508
 #if DM_VERSION < MIN_COMPILER_VERSION //Update this whenever you need to take advantage of more recent byond features
-#error Your version of BYOND is too out-of-date to compile this project. Go to byond.com/download and update.
+#error Your version of BYOND is too out-of-date to compile this project. Go to byond.com/download and update. As for August of 2015, it uses the BETA version of BYOND.

```

Comment:
Would it not go through the preprocessors first and receive that particular error before attempting to compile?